security/fma-operator-fee.md

```diff
@@ -122,17 +163,45 @@ Below are references for this project:
+
+    > `gas` is a uint64, so `(gasUsed * operatorFeeScalar / 1e6) + operatorFeeConstant` can be at most `(u64.max * u32.max / 1e6) + u64.max ~= 7.924660923989131e+22`, which is an int of bit length 77 and fits comfortably within a uint256 variable allocation.
+
+- **Detection:**
+  - NAT fuzz testing (TODO)
+- **Recovery Paths(s):**
+  - A bug within the OP EVM would critical and would require an emergency upgrade of the sequencer and bridge.
+
+
 ## Action Items
 
 Below is what needs to be done before launch to reduce the chances of the above failure modes occurring, and to ensure they can be detected and recovered from:
 
+ 
 - [ ] Coordinate with wallet providers to update their fee estimation logic
 - [ ] Implement automated monitoring on dabase growth rate
 
+**Testing**
+- [ ] (BLOCKING): **NAT tests** with Kurtosis
+    - [ ] A force inclusion transaction from L1.
+    - [ ] A case with EIP7702. Since now EOA can have code, we need to check whether transactions without `msg.sender`, `tx.origin` leads to unexpected behavior
+- [ ] (BLOCKING): ***Differential Fuzzing*** with op-reth/op-geth to avoid any chain-split on the operator fee component (with the refund).
+
+**Monitoring:** 
+- [ ] (NON-BLOCKING): **Monitoring** a conservation of balance invariant. If the invariant is broken it should immediately raise alerts.
+  - [**PR:** feat(monitorism): Add ETH conservation invariant monitor](https://github.com/ethereum-optimism/monitorism/pull/135)
+
 ## Audit Requirements
 
-An audit has not been deemed necessary for the relatively simple changes to the SystemConfig contract, which has been reviewed by Base internal security team.
+An audit has not been deemed necessary on **L1** for the relatively simple changes to the SystemConfig contract, which has been reviewed by Base internal security team. 
 
 Indeed, the only addition is the `setOperatorFeeScalars` function, which is a setter function that updates the operator fee parameters and trigger an event. This function is callable by the SystemConfig owner only.
 
-Additionally, we are performing multi-client testing (op-geth and op-reth) and will run a multi-client devnet, so bugs in either will be quickly detected by consensus disagreements among them. So, we don't think the operator fee feature falls into the [Audit Framework](https://gov.optimism.io/t/op-labs-audit-framework-when-to-get-external-security-review-and-how-to-prepare-for-it/6864) Existential + Safety category.
\ No newline at end of file
+However, for the **clients code** (op-geth/op-reth) and not the smart contracts: Since this is modifiying **the state transition**, @Ethnical recommends an audit of this feature, however they would need to see with other member of evm-safety to know what they think etc). Following a conversation with @clabby, @tynes, @teddyknox and @Ethnical the decision was made not to block the release of Isthmus on an audit of the op-geth logic touched.
```

Comment:
```suggestion
An audit has not been deemed necessary right now of this feature. 
For more context, the feature is about to be used by a low amount of Chain Operator in the near future.  
Following a conversation with @clabby, @tynes, @teddyknox and @Ethnical the decision was made not to block the release of Isthmus on an audit of the `op-geth` logic touched.
However, to make an audit in parallel of the implementation of the feature in case in near feature of an wide adoption of this feature. 
To recap most of the current chains would not have this feature enabled.  
```